collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -51,21 +60,26 @@ def _get_data(self):
         alarms = raw_data.get('alarms', {})
 
         data = {a: self.sm[alarms[a]['status']] for a in alarms if alarms[a]['status'] in self.sm}
-        self.update_charts(alarms, data)
+        self.update_charts('alarms', data)
         data['alarms_num'] = len(data)
 
+        if self.collect_alarm_values:
+            data_values = {'{}_value'.format(a): alarms[a]['value'] * 100 for a in alarms if 'value' in alarms[a] and alarms[a]['value'] is not None}
+            self.update_charts('values', data_values, divisor=100)
+            data = {**data, **data_values}
```

Comment:
```suggestion
            data.update(data_values)
```

py2 is dead, but it is still deafult on some old distros. How about to use dict.update?